CONTRIBUTING.md

```diff
@@ -13,6 +13,24 @@ How to contribute
 -  **Check good first-issue** - check out [good first issues](https://github.com/kairos-io/kairos/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) if you want to contribute to a specific problem
 -  **Read about our code of conduct and governance**: Kairos is an Open source, community-driven project with a [governance](https://github.com/kairos-io/kairos/blob/master/GOVERNANCE.md) and adopts CNCF [Code of conduct](https://github.com/kairos-io/kairos/blob/master/CODE_OF_CONDUCT.md)
 
+Guiding user stories
+--------------------
+
+These "guiding user stories" summarize the scope and ideals of the Kairos project. These are all stories that can grow with context: every new device or OS that Kairos supports can have an implication for each of these stories. We cannot ask every contribution to satisfy them all at once. Instead, we ask that each new feature or capability should fit into at least one guiding user story and that every lower-numbered user story should be satisfied first.
+
+1. I can install k8s on an experimental device.
```

Comment:
Does k8s mean any version of Kubernetes, including k3s? Would it be better to say k3s, since that's what is provided? Or if we want to keep it open, maybe use the word Kubernetes? Please excuse the n00b question, I'm very new to the Kubernetes community, so I'm not sure what people understand when they read k8s.